### PR TITLE
Add queue pause enabled status attribute

### PR DESCRIPTION
**Pull request by stcheng**
Signed-off-by: Shuotian Cheng <shuche@microsoft.com>

### CODE REVIEW

**Comment by kcudnik** [comment]
Enums must be sorted, thats why build fails

**Comment by stcheng** [comment]
@kcudnik So If I will need to add some more READ_ONLY attributes, the rest enums need to be renumbered. This will fail the backward compatibility?

**Comment by kcudnik** [comment]
No, just add them at the end, w don't need to stick to rule that all read only attributes must be at the top

**Comment by stcheng** [comment]
So, the comment flag: /* READ_ONLY */ and /* READ & WRITE */ make no sense in the future.

**Comment by kcudnik** [comment]
Yes, we can remove them

**Comment by stcheng** [comment]
updated

File: inc/saiqueue.h [hunk 1]
```diff
@@ -118,8 +118,7 @@ typedef enum _sai_queue_attr_t
     SAI_QUEUE_ATTR_WRED_PROFILE_ID = 0x00000004,
 
     /**
-     * @brief Attach buffer profile to Queue
-     * Default no profile
```
Comment (atitjain) [inline inc/saiqueue.h]: is the default taken out?

**Pull request merged**